cocos/ui/UIScrollView.cpp

```diff
@@ -1342,8 +1342,8 @@ void ScrollView::copySpecialProperties(Widget *widget)
     if (scrollView)
     {
         Layout::copySpecialProperties(widget);
-        _innerContainer = scrollView->_innerContainer;
         setDirection(scrollView->_direction);
+        setInnerContainerPosition(scrollView->getInnerContainerPosition());
```

Comment:
thanks @Almax27 
@pandamicro  The `copySpecialProperties` method is a protected method and the `_innerContainer` is initialized with the default settings, there is no need to copy the properties.
Since when ScrollView is created, the `_innerContainer` already has the default settings and we should only clone the children which is done by `ScrollView::copyClonedWidgetChildren`.
So your PR is good to merge.